### ISSUE TITLE
volcanoly appears to disregard effect_size argument passed via ...

### ISSUE THREAD

**Issue by rpkyle** [sahirbhatnagar/manhattanly#21]
The `dashBio` package for R implements wrappers for `manhattanly` and `volcanoly`, to collapse the formal arguments for both functions with those from `manhattanr` and `volcanor`, while using the same (keyword) arguments as in the Python implementation in [plotly/dash-bio](https://github.com/plotly/dash-bio).

An app developer recently noted that the `dashbioVolcano` wrapper seemed to ignore the `effect_size` parameter completely. 

The `volcanor` function does accept this parameter, and uses it when constructing an internal `data.frame`:

https://github.com/sahirbhatnagar/manhattanly/blob/f14e539630b1667437f5c254eb2631785f8f29e5/R/volcanor.R#L95-L96

However, the `volcanoly` functions [here](https://github.com/sahirbhatnagar/manhattanly/blob/master/R/volcanoly.R) don't appear to accept an `effect_size` parameter (although they accept `...`, this appears to be unused within the body of `volcanoly.volcanor` -- `x` is passed to `mh`, but I couldn't find `...` inside the method when I searched):

https://github.com/sahirbhatnagar/manhattanly/blob/f14e539630b1667437f5c254eb2631785f8f29e5/R/volcanoly.R#L116-L117

https://github.com/sahirbhatnagar/manhattanly/blob/f14e539630b1667437f5c254eb2631785f8f29e5/R/volcanoly.R#L137-L153

@sahirbhatnagar Is there a "right" way to pass `effect_size` to `volcanoly`? I imagine I'm doing it improperly, but wanted to mention ☝️ in case this behaviour is not intentional.

The code for my `dashbioVolcano` wrapper is [located here](https://github.com/plotly/dash-bio/blob/master/R/dashbioVolcano.R).

**Comment by sahirbhatnagar** [comment]
Thanks for reporting this. The issue was due to the misplacement of the ellipsis `...`. Due to partial argument matching, the `effect_size` argument was never getting passed to `volvanor`. As mentioned in this [SO post](https://stackoverflow.com/a/14335647/3285322), I should have placed all arguments for exact matching after the `...` This issue has now been fixed in the most recent release:

```r
# trying different columnnames, since default is "EFFECTSIZE"
HapMap2 <- HapMap
colnames(HapMap2) <- c("chromosome","baseposition","Pval","rs","zvalue", "logFC","taxa","dist")
p <- volcanoly(HapMap2, 
               p = "Pval",
               effect_size = "logFC",
               snp = "rs",
               gene = "taxa")
``` 

See the following link for the resulting plot:
https://plotly.com/~sahirbhatnagar/151/

**Comment by sahirbhatnagar** [comment]
Fixed with #24

**Comment by rpkyle** [comment]
Thanks @sahirbhatnagar!
